File: src/main/java/com/thunderbear06/sounds/SoundRegistry.java
package com.thunderbear06.sounds;

import com.thunderbear06.CCAndroids;
import net.minecraft.sounds.SoundEvent;
import net.neoforged.neoforge.registries.DeferredHolder;

public final class SoundRegistry {
    public static final DeferredHolder<SoundEvent, SoundEvent> ANDROID_AMBIENT = CCAndroids.ANDROID_AMBIENT;
    public static final DeferredHolder<SoundEvent, SoundEvent> ANDROID_HURT = CCAndroids.ANDROID_HURT;
    public static final DeferredHolder<SoundEvent, SoundEvent> ANDROID_DEATH = CCAndroids.ANDROID_DEATH;

    private SoundRegistry() {
    }
}

File: src/main/kotlin/com/thunderbear06/sounds/SoundRegistry.kt
package com.thunderbear06.sounds

import com.thunderbear06.CCAndroids
import net.minecraft.sounds.SoundEvent
import net.neoforged.neoforge.registries.DeferredHolder

object SoundRegistry {
    @JvmField val ANDROID_AMBIENT: DeferredHolder<SoundEvent, SoundEvent> = CCAndroids.ANDROID_AMBIENT
    @JvmField val ANDROID_HURT: DeferredHolder<SoundEvent, SoundEvent> = CCAndroids.ANDROID_HURT
    @JvmField val ANDROID_DEATH: DeferredHolder<SoundEvent, SoundEvent> = CCAndroids.ANDROID_DEATH
}
